README.md

```diff
@@ -8,12 +8,13 @@ To learn more about the University of Belize Environmental Research Institute, n
 
 ## Authors:
 Jessica C. Boles, Data Scientist, University of Belize Environmental Research Institute
+Justin Brodhurst, Student, University of Belize
```

Comment:
Could you please ensure each author displays on a different line? I think it will need to be double spaced.
If you press control-shift-v while in the .md file, it will open up a preview window so you can see how it will look

Reply:
Didn't know about that shortcut, Added backslashes as they dont leave a line space between each.